assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -18,20 +18,25 @@ import '../../shared/data/api-fetch-preloaded-once';
  * Editor wizard modal component.
  */
 const EditorWizardModal = () => {
-	const dataState = useState( {} );
+	const wizardDataState = useState( {} );
 	const { editPost, savePost } = useDispatch( editorStore );
 
 	const [ open, setDone ] = useWizardOpenState();
 	const steps = useEditorWizardSteps();
 	const setDefaultPattern = useSetDefaultPattern( {
-		'sensei-content-description': dataState[ 0 ].courseDescription,
+		'sensei-content-description': wizardDataState[ 0 ].description,
```

Comment:
Since we're accessing the `wizardDataState[ 0 ]` many times now, we could make it clearer by creating a new variable like `const wizardData = wizardDataState[ 0 ];`. WDYT?